iCubTelemVizServer/devices/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the Apache-2.0 license
+
+cmake_minimum_required(VERSION 3.5)
+project(matlab-whole-body-simulator)
+
+# There is nothing to build, only the fake battery config files need to be copied to the install folder:
+
+set(YARP_ROBOTS_INSTALL_DIR "share/yarp/robots" CACHE
+    STRING "Location (relative to the install prefix) where the robot configuration files are installed.")
```

Comment:
Since ``yarp`` is for sure a dependency, you can use a machinery like https://github.com/robotology/whole-body-controllers/blob/c5067a8e304b85c846c3a16463200de368b9a608/CMakeLists.txt#L14-L15.

This will create some CMake variable that will avoid hardcoding this.

See for example https://github.com/robotology/whole-body-controllers/blob/c5067a8e304b85c846c3a16463200de368b9a608/utilities/homePositions/robots/iCubGazeboV2_5/CMakeLists.txt

Reply:
Thanks for the hint @S-Dafarra . I looked into the YARP documentation ["Configuring your external build"](http://www.yarp.it/git-master/yarp_external_build.html) and some documentation on the command [`FILE(GLOB ...)`](https://cmake.org/cmake/help/latest/command/file.html#glob).

So, following your suggestion, I've used the Yarp helper `yarp_configure_external_installation` to define the destination folder `YARP_ROBOTS_INSTALL_DIR`. I tried using the `yarp_install` method, but it seemed less convenient to me for the following reasons:
- it requires getting the source files via `file(GLOB ...)`, and apparently, when source files are collected through globbing CMake doesn't detect properly changes on those files and when to re-install them. I haven't checked this.
> Note
> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The CONFIGURE_DEPENDS flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild. 
- `yarp_install` takes as input the list of source files, not a source directory, and unlike `install`, does not conserve the tree structure of the installed files.

If we don't use `yarp_install`, we loose the check done by that function on the destination directory, with the recommendation to update the YARP_DATA_DIRS, but it''s something we can recommend in the README anyway.